### ISSUE TITLE
Remove unsupported Webmin modules

### ISSUE THREAD

**Issue by jcameron** [webmin/webmin#1950]
@iliajie @swelljoe as discussed, I think we can remove the following : 

wuftpd - Seems to be no longer maintained, and ProFTPd is way better

mon - Pretty sure this is no longer developed

jabber - The 1.x version we support has been deprecated, and version 2.x is a completely different product

tcpwrappers - I think this is no longer developed, but hard to be sure

**Comment by iliajie** [comment]
Hey, Jamie.

> tcpwrappers - I think this is no longer developed, but hard to be sure

I think it's still used and works on Debian/Ubuntu, i.e. SSH is still compiled against the `libwrap.a` library, e.g.:

```
root@ubuntu22-pro:~# ldd /usr/sbin/sshd | grep libwrap
libwrap.so.0 => /lib/x86_64-linux-gnu/libwrap.so.0 (0x00007fee89c7b000)
```

The other mentioned modules should go, I agree.

Also what about:

  `grub` - no one uses Grub (1) anymore, right? Our module doesn't seem to work on modern systems
  `heartbeat` - is available only on Debian/Ubuntu but it's not working out of the box. Not sure if module still would work with it
  `vgetty` - is it still a thing?
   `qmail` - hasn't it been unmaintained for decades? Same with `openslp`, right?
   `krb5` - these are also seem old? Also, may be `ipsec`, `idmapd, and `adsl-client`
   
And those PPP packages, i.e. `pap`, `ppp-client`, `pptp-client` and `pptp-server`

**Comment by swelljoe** [comment]
`vgetty` has, as far as I know, been completely replaced by Asterisk by anyone who needs voice services. And that happened a decade or two ago.

I can't find any evidence `vgetty` is actively maintained, and the most recent docs I can find are from 2003. The most recent mention I can find in relation to `vgetty` in Webmin is from 2002: https://sourceforge.net/p/webadmin/support-requests/15/

If anyone is using `vgetty` today they're being very quiet about it.

**Comment by swelljoe** [comment]
If we don't support GRUB2, we shouldn't be offering a GRUB module at all. GRUB-Legacy is long unmaintained and all distros switched to GRUB2 well over a decade ago.

There are a few forks of `qmail` that remain compatible, AFAIK, and at least a couple are still lightly maintained. https://en.wikipedia.org/wiki/Qmail

I think qmail can stay, but only if we know it works with those forks.

Kerberos is still widely used at universities, and krb5 is the currently maintained version (most recent release was just a couple months ago). http://web.mit.edu/kerberos/

I don't know if we have any users using the Webmin module, but it's not in the same category as all this other stuff; it is well-maintained software. I see people talking about it on Mastodon pretty regularly, even (people at universities).

**Comment by jcameron** [comment]
Good point, the Grub module seems obsolete nowdays. Basically nobody needs to edit kernel parameters anymore, because it's taken care of by the package system. Also I would agree that we can drop vgetty.
